examples/microprofile/static-content/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -16,7 +16,7 @@
 
 # web server configuration
 # Use a random free port
-server.port=0
+server.port=8080
```

Comment:
Is that intentional?

Reply:
this is an example, where random port is not desired.